### PR TITLE
adjustments for fault state for debugging

### PR DESCRIPTION
**Pull request by connorl309**
Replace RTOS and Task stuff regarding FaultState to directly call the function for easier debugging purposes

# Quality Assurance Checklist 
To make reviews more efficient, please make sure the software feature meets the following standards and check everything off that meets the quality check. Once everything has been checked, the assigned reviewers will begin the review process. _Edit this description to check off the list._

There are exceptions with all guidelines. As long as your decisions are justified, then you are good! Contact the reviewers or the leads about any exceptions. 

## Requirements 
- [ ] Followed Coding Style Guide 
- [ ] Code Build checks pass 
- [ ] No merge conflicts 
- [ ] Software feature has test associated with it. 
- [ ] Software feature has documentation for it updated.
- [ ] Test provides useful information and uses relevant data to accurately represent BPS 
    - NOTE: If test file already exists, use that one 
- [ ] 2 Members of the team have already reviewed my PR 
    - NOTE: Approver will not look at PR if 2 other members haven't reviewed first 
- [ ] Did you Have Fun?

## Things to Consider 
- [ ] Even if the above are checked, is this the best way of writing my code? 
    - It's possible to write code that works, but are there ways to make code more efficient?

### CODE REVIEW

File: Tasks/Inc/Tasks.h [hunk 1]
```diff
@@ -45,7 +45,7 @@ enum WDogBits_e {
 
 void Task_Init(void *p_arg);
 
-void Task_FaultState(void *p_arg);
+void EnterFaultState(); // here despite not being a task, so compilation works
```
Comment (manthanand) [inline Tasks/Inc/Tasks.h:48]: move it somewhere else

File: Tests/Test_CriticalState_1.c [hunk 2]
```diff
@@ -52,14 +52,15 @@ void Task1(void *p_arg){
     RTOS_BPS_SemCreate(&Fault_Sem4, "Fault/Tripped Semaphore", 0);
 
     RTOS_BPS_SemCreate(&SafetyCheck_Sem4, "Safety Check Semaphore", 0);
-
+    /*
```
Comment (manthanand) [inline Tests/Test_CriticalState_1.c:55]: Delete this

File: Tests/Test_FaultState.c [hunk 3]
```diff
@@ -35,6 +35,7 @@ void Task1(void *p_arg){
 
     OS_CPU_SysTickInit(SystemCoreClock / (CPU_INT32U) OSCfg_TickRate_Hz);
 
+    /*
```
Comment (manthanand) [inline Tests/Test_FaultState.c:38]: delete this

File: Tests/Test_SafetyCheck.c [hunk 4]
```diff
@@ -239,14 +239,15 @@ int main() {
     // assert
 
     BSP_UART_Init(foo, foo, UART_USB);
-
+    /*
```
Comment (manthanand) [inline Tests/Test_SafetyCheck.c:242]: delete this

File: Tests/Test_VoltTempMonitorTask.c [hunk 5]
```diff
@@ -58,13 +58,15 @@ void Task1(void *p_arg){
 
     // Spawn tasks needed for Amperes readings to affect contactor
     //1
+    /*
```
Comment (manthanand) [inline Tests/Test_VoltTempMonitorTask.c:61]: delete this

**Review by ray4477 — COMMENTED** [review]
if ur trying to merge into master wouldn't you need checks for DEBUG flag in pre processing. if its a sep branch just for debugging its fine though.

**Comment by connorl309** [comment]
> if ur trying to merge into master wouldn't you need checks for DEBUG flag in pre processing. if its a sep branch just for debugging its fine though.

No this isn't for like debug compilation stuff. It's to remove the RTOS nastiness when running GDB on the real board

File: Tests/Test_Amperes_Task.c [hunk 6]
```diff
@@ -59,13 +59,16 @@ void Task1(void *p_arg){
 
     // Spawn tasks needed for Amperes readings to affect contactor
     //1
+    /*
```
Comment (manthanand) [inline Tests/Test_Amperes_Task.c:62]: delete this too

**Review by manthanand — APPROVED** [review]

**Pull request merged**